src/main/resources/english.yml

```diff
@@ -250,6 +250,10 @@ msg_siege_war_banner_control_session_started_with_altitude: '&bBanner control se
 msg_siege_war_banner_control_session_failure: '&cBanner control session failed!. Common causes for this include: dying, going too far from the banner, going into the town, or flying.'
 msg_siege_war_banner_control_session_failure_with_altitude: '&cBanner control session failed!. Common causes for this include: dying, going too far from the banner, going below the altitude of the banner, going into the town, or flying.'
 
+#Unsiegeable towns strings
+msg_permanent: "permanent"
+msg_error_must_be_num_or_permanent: '&cAmount must be a number or ''permanent''.'
+

```

Comment:
:construction: 
- The english.yml file now needs a little rearranging
- Please make the following small updates:
  - Update the version at the top of the file to `23`
  - Add a new comment line at the bottom of the file called `#Added in 23`
  - Move all the new lines under the new comment line